packages/cli/src/domains/magento2/functions/checkYarn.ts

```diff
@@ -0,0 +1,24 @@
+import { spawn } from 'child_process';
+import { logSimpleErrorMessage, simpleLog } from './terminalHelpers';
+
+/** Checking if Docker is installed and running on user's machine */
+const checkYarn = async (): Promise<void> => {
```

Comment:
This can be checked automatically with `engines` in `package.json` same as Node.js version.

Reply:
I may get it wrong, but I guess that setting it up in package json will affect the running of CLI in general. But I need to check that Yarn exists in order to install dependencies after the project is generated. 

So that user will only run `yarn dev`